compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/viewinterop/InteropViewHolder.skiko.kt

```diff
@@ -97,17 +118,14 @@ internal open class InteropViewHolder(
     }
 
     override fun onReuse() {
-        isAttachedToWindow = true
         reset()
     }
 
     override fun onDeactivate() {
-        isAttachedToWindow = false
     }
 
     override fun onRelease() {
         release()
-        isAttachedToWindow = false
```

Comment:
The change is not clear.
1. I think it should remain private
2. It's not clear how this will be implemented on Android (it's supposed to be reused, right?)

Reply:
On Android it calls immediately to `View.isAttachedToWindow` inside `AndroidViewHolder`. Since the logic that relies on this flag is reused, it makes sense for platform holders to implement it.

It's either this, or abstract protected val.

Reply:
On Android it's this:

```
    override fun onAttachedToWindow() {
        super.onAttachedToWindow()
        runUpdate()
    }

    override fun onDetachedFromWindow() {
        super.onDetachedFromWindow()
        // remove all observations:
        snapshotObserver.clear(this)
    }
```

Reply:
Refactored it to make `InteropViewHolder` responsible for proper hierarchy maintenance